plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Customize the ISO file by add/delete/modify files
+description:
+  - This module is used to customize the ISO file by add/delete/modify files.
+author:
+  - Yuhua Zou (@ZouYuhua) <zouy@vmware.com>
+requirements:
+  - "pycdlib"
+  - "python >= 2.7"
+version_added: '5.5.0'
+
+options:
+  src_iso:
+    description:
+    - This is the path of source ISO file.
+    - Will fail if specified ISO file does not exist on local machine.
+    - 'Note: With all ISO9660 levels from 1 to 3, all file names are restricted to uppercase letters, numbers and
+      underscores (_). File names are limited to 31 characters, directory nesting is limited to 8 levels, and path
+      names are limited to 255 characters.'
+    type: str
+    required: yes
+  dest_iso:
+    description:
+    - The path with file name of the customized ISO file on local machine.
+    - Will create intermediate folders when they does not exist.
+    type: str
+    required: yes
+  add_files:
+    description:
+    - The absolute path with file name on ISO file.
+    - Will create intermediate folders when they does not exist.
+    - The file will be replaced if already exists in ISO.
```

Comment:
This sounds somewhat confusing. How about:
```suggestion
    - Allows to add and replace files in the ISO file.
    - Will create intermediate folders inside the ISO file when they do not exist.
```

Reply:
Fix it. Thanks